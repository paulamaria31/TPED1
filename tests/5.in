200
XXVOXVVVO
OOVVXXXXX
XXVXXXOXO
VXOXOVXXV
XOXVOVOVO
VXVXXXVXV
OOVVVVVVV
OOXOXOVVX
OOVXXOOXO
XVXVOVXXV
VOXVXXXXX
OVXOOVVVO
VVXOXXOVO
VVOOOVVVV
OOXOVOVVO
OXVVVVXVO
VVOXVOXVX
OVOXOXXOX
VOVOXXXOV
VVOXVVOOV
VXVXOVOOO
OOVXOVXVV
XXVOOVOOV
OXOXVOOVV
OXXVOOVXO
XXOVOVXVO
XVXVOXOXV
XVXVVOXVO
OXVVXVXXV
OVVXOVVOV
XVOVXXVVV
OXVXVVVVO
VOXVXOVOV
XOXXVVXVX
OVXOOVOXO
XVOVXXVVV
OOXXXVXXX
VOVOXOXXV
OOXOXVVOV
VOOVXXOVO
VVOOXOOXX
XOVOVVOXV
VXOVOXOOX
VXVXXXVVO
XOXOXOOXO
XOOOVVOVO
OOOXVOOOV
VXXVVVXVX
OOVXXXVOX
OXVVXXVXV
VOXOVVOVO
XVVXVVXXX
OXVVVXOVO
VOVVVXOXV
OOVOOVXXX
XXOVOXVVX
XVVXOVOVV
OXVVVXVOV
XVOOXVOXO
XOOVVOOVX
VXXXOVVVV
XXXXOVVXV
XOXXVOVOV
XOOVXVXXO
OVOXVVOXX
VVXVOOXVV
OVXOVVXVX
VVOXVXOXX
XVVOOXOOX
OXVVOOXXV
XXXVVOOVV
XXVOOOOOO
VXXOOVXOX
XVXOXOVOV
VXVOVVOVO
OOOVOOVVO
OOOXXXVVO
VXXXVXVVX
XOOVXOOOV
XVOXOXOVV
VOOXXXVXX
OVVOVXXOV
OOVVOOXOV
VXXXVVOXO
XOOXVOVVV
VVOOOXXOX
VOVOVVXXO
OVVXXOVVX
XVVVOXXVO
XVVVXVXXV
XVOVVXOVX
XOOXXOOVX
VXOOVXXXX
VXXXOXXOV
VXOVOOOVX
OVXOVOVXO
XXVOOVXVV
VOXOXOVVX
VOVOVOOVX
VXXXOVOVV
OOVVVXVVX
VXXVVVVXO
VVVXVXVOO
OOVOOOVXO
XXVXOVVVV
OXXOXOXXX
OOOVXOVXX
XXXXOXOVX
VVXVVOXOO
VOXVOVXVO
VXOXXXOXV
XXVXVXOOO
XVVVXVOOX
OVOXOXVXV
XOXOVOOXV
XVVVOOVVV
XVVXOXXXO
VXXOOVXOV
XVVOXOVOV
XXVOXOVXX
XOVOOVOVX
XVOOOOXXO
XOXVVVXXX
OOVOOOOXX
XVVXVXOXO
XXXOOXOOO
XOXOOVVVX
OXVVXVOOV
VXVXXOOOO
OVVOVOXXO
XVVOVOOOO
VXOXVVOXV
VXVOVOXOO
VVXOXXOVO
VOOOOVOVV
OXOVXOXVO
XVXVOOOXV
VOOOOXOXV
OVXXVOVOV
OXVVOVXVO
OXVVOOOXV
VOXXXOXVO
OOVOXVVOX
VXXOXXVVX
OXOOVXXXO
OOOVXXOOX
XVVXXXXVO
OVOVVOXXV
XXOOVOOXX
VOXVOXVXO
XOXVVOOOX
VOOVOXXXV
OOXVXOVVV
OXOOVXOOX
VOVOOXXVV
XXXOXXVVV
XVXVOXVVV
XVOXOXVOV
OXXVVOXXV
OOOVVOXOX
XOOOVOXVO
OOVXXXXVX
XVXXOVVVO
XVOOVVVVV
VOOXVVOVX
OVOVOOVVO
VVVOVOOVO
XVXOXVOVX
XXXXXXXOV
OOXVVOVXX
VOXXOXXXO
OOVVVXXVX
VOVXVOVVV
XOXOOOVOO
OOOXVVVVX
OXOXXVOXX
OOVXXXOVO
XVOXOXOXX
OOVVVOXXX
OXVVOVXXX
VOVOVOOXO
OXOXXVOXV
OOOXOXOOO
VVOXVXXVO
OOVOXOXOX
XOXVVXVOX
VVXVXOVXX
VOXXOVVOV
VOOVXOOVV
OVXVXVOOO
VVVXVOOXO
VXOOXXOOV
XXVOXVXVV
OVOVVVXVV
XXOVVVXOO
VOXXOXOVV
OVVXOVVVV
VOOXOXVVX
OVOXVVOXV
VXOVXVXXO
