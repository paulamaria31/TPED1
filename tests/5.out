Tabuleiro 1 em andamento [O: sem jogada mestre]
Tabuleiro 2 invalido
Tabuleiro 3 invalido
Tabuleiro 4 invalido
Tabuleiro 5 invalido
Tabuleiro 6 invalido
Tabuleiro 7 invalido
Tabuleiro 8 em andamento [X: (3,1)]
Tabuleiro 9 invalido
Tabuleiro 10 invalido
Tabuleiro 11 invalido
Tabuleiro 12 invalido
Tabuleiro 13 em andamento [proximo jogador indefinido]
Tabuleiro 14 invalido
Tabuleiro 15 invalido
Tabuleiro 16 em andamento [proximo jogador indefinido]
Tabuleiro 17 em andamento [O: sem jogada mestre]
Tabuleiro 18 em andamento [proximo jogador indefinido]
Tabuleiro 19 em andamento [proximo jogador indefinido]
Tabuleiro 20 invalido
Tabuleiro 21 invalido
Tabuleiro 22 em andamento [X: sem jogada mestre]
Tabuleiro 23 invalido
Tabuleiro 24 invalido
Tabuleiro 25 com vitoria [O]
Tabuleiro 26 em andamento [proximo jogador indefinido]
Tabuleiro 27 invalido
Tabuleiro 28 em andamento [O: sem jogada mestre]
Tabuleiro 29 invalido
Tabuleiro 30 invalido
Tabuleiro 31 invalido
Tabuleiro 32 em andamento [proximo jogador indefinido]
Tabuleiro 33 em andamento [X: (3,1)]
Tabuleiro 34 invalido
Tabuleiro 35 invalido
Tabuleiro 36 invalido
Tabuleiro 37 invalido
Tabuleiro 38 em andamento [proximo jogador indefinido]
Tabuleiro 39 invalido
Tabuleiro 40 invalido
Tabuleiro 41 em andamento [X: (1,2)(1,1)]
Tabuleiro 42 em andamento [X: sem jogada mestre]
Tabuleiro 43 com vitoria [O]
Tabuleiro 44 invalido
Tabuleiro 45 deu velha
Tabuleiro 46 invalido
Tabuleiro 47 invalido
Tabuleiro 48 invalido
Tabuleiro 49 com vitoria [X]
Tabuleiro 50 invalido
Tabuleiro 51 invalido
Tabuleiro 52 invalido
Tabuleiro 53 em andamento [X: sem jogada mestre]
Tabuleiro 54 em andamento [proximo jogador indefinido]
Tabuleiro 55 com vitoria [X]
Tabuleiro 56 invalido
Tabuleiro 57 em andamento [proximo jogador indefinido]
Tabuleiro 58 em andamento [proximo jogador indefinido]
Tabuleiro 59 em andamento [X: (1,2)]
Tabuleiro 60 invalido
Tabuleiro 61 invalido
Tabuleiro 62 invalido
Tabuleiro 63 em andamento [proximo jogador indefinido]
Tabuleiro 64 em andamento [O: (2,3)]
Tabuleiro 65 em andamento [proximo jogador indefinido]
Tabuleiro 66 em andamento [proximo jogador indefinido]
Tabuleiro 67 em andamento [O: sem jogada mestre]
Tabuleiro 68 invalido
Tabuleiro 69 em andamento [X: (1,3)]
Tabuleiro 70 em andamento [proximo jogador indefinido]
Tabuleiro 71 com vitoria [X]
Tabuleiro 72 invalido
Tabuleiro 73 em andamento [O: (2,3)]
Tabuleiro 74 em andamento [proximo jogador indefinido]
Tabuleiro 75 invalido
Tabuleiro 76 invalido
Tabuleiro 77 invalido
Tabuleiro 78 invalido
Tabuleiro 79 invalido
Tabuleiro 80 com vitoria [O]
Tabuleiro 81 invalido
Tabuleiro 82 em andamento [X: sem jogada mestre]
Tabuleiro 83 invalido
Tabuleiro 84 invalido
Tabuleiro 85 em andamento [X: (3,1)]
Tabuleiro 86 em andamento [X: sem jogada mestre]
Tabuleiro 87 em andamento [X: sem jogada mestre]
Tabuleiro 88 em andamento [O: sem jogada mestre]
Tabuleiro 89 em andamento [O: sem jogada mestre]
Tabuleiro 90 invalido
Tabuleiro 91 em andamento [O: (2,2)]
Tabuleiro 92 com vitoria [X]
Tabuleiro 93 invalido
Tabuleiro 94 invalido
Tabuleiro 95 invalido
Tabuleiro 96 invalido
Tabuleiro 97 em andamento [O: (2,3)]
Tabuleiro 98 em andamento [proximo jogador indefinido]
Tabuleiro 99 invalido
Tabuleiro 100 em andamento [O: sem jogada mestre]
Tabuleiro 101 em andamento [proximo jogador indefinido]
Tabuleiro 102 invalido
Tabuleiro 103 em andamento [proximo jogador indefinido]
Tabuleiro 104 invalido
Tabuleiro 105 invalido
Tabuleiro 106 invalido
Tabuleiro 107 com vitoria [O]
Tabuleiro 108 invalido
Tabuleiro 109 em andamento [X: (2,2)]
Tabuleiro 110 em andamento [X: sem jogada mestre]
Tabuleiro 111 invalido
Tabuleiro 112 com vitoria [O]
Tabuleiro 113 com vitoria [X]
Tabuleiro 114 em andamento [proximo jogador indefinido]
Tabuleiro 115 em andamento [X: sem jogada mestre]
Tabuleiro 116 em andamento [X: sem jogada mestre]
Tabuleiro 117 invalido
Tabuleiro 118 em andamento [proximo jogador indefinido]
Tabuleiro 119 em andamento [X: (3,3)]
Tabuleiro 120 invalido
Tabuleiro 121 invalido
Tabuleiro 122 invalido
Tabuleiro 123 invalido
Tabuleiro 124 invalido
Tabuleiro 125 invalido
Tabuleiro 126 invalido
Tabuleiro 127 em andamento [proximo jogador indefinido]
Tabuleiro 128 em andamento [X: sem jogada mestre]
Tabuleiro 129 com vitoria [O]
Tabuleiro 130 invalido
Tabuleiro 131 invalido
Tabuleiro 132 em andamento [O: (2,2)]
Tabuleiro 133 invalido
Tabuleiro 134 em andamento [proximo jogador indefinido]
Tabuleiro 135 invalido
Tabuleiro 136 com vitoria [O]
Tabuleiro 137 em andamento [proximo jogador indefinido]
Tabuleiro 138 invalido
Tabuleiro 139 em andamento [X: sem jogada mestre]
Tabuleiro 140 com vitoria [O]
Tabuleiro 141 invalido
Tabuleiro 142 com vitoria [X]
Tabuleiro 143 invalido
Tabuleiro 144 invalido
Tabuleiro 145 em andamento [proximo jogador indefinido]
Tabuleiro 146 invalido
Tabuleiro 147 invalido
Tabuleiro 148 em andamento [X: (3,3)]
Tabuleiro 149 em andamento [proximo jogador indefinido]
Tabuleiro 150 em andamento [proximo jogador indefinido]
Tabuleiro 151 em andamento [X: (2,2)]
Tabuleiro 152 em andamento [proximo jogador indefinido]
Tabuleiro 153 em andamento [X: (3,1)]
Tabuleiro 154 invalido
Tabuleiro 155 em andamento [X: sem jogada mestre]
Tabuleiro 156 invalido
Tabuleiro 157 invalido
Tabuleiro 158 em andamento [proximo jogador indefinido]
Tabuleiro 159 invalido
Tabuleiro 160 invalido
Tabuleiro 161 invalido
Tabuleiro 162 invalido
Tabuleiro 163 em andamento [O: sem jogada mestre]
Tabuleiro 164 em andamento [X: sem jogada mestre]
Tabuleiro 165 em andamento [X: sem jogada mestre]
Tabuleiro 166 invalido
Tabuleiro 167 invalido
Tabuleiro 168 invalido
Tabuleiro 169 invalido
Tabuleiro 170 em andamento [proximo jogador indefinido]
Tabuleiro 171 invalido
Tabuleiro 172 em andamento [O: (1,3)]
Tabuleiro 173 em andamento [X: sem jogada mestre]
Tabuleiro 174 invalido
Tabuleiro 175 com vitoria [O]
Tabuleiro 176 invalido
Tabuleiro 177 com vitoria [X]
Tabuleiro 178 invalido
Tabuleiro 179 com vitoria [X]
Tabuleiro 180 invalido
Tabuleiro 181 invalido
Tabuleiro 182 com vitoria [X]
Tabuleiro 183 invalido
Tabuleiro 184 em andamento [O: sem jogada mestre]
Tabuleiro 185 invalido
Tabuleiro 186 invalido
Tabuleiro 187 invalido
Tabuleiro 188 com vitoria [O]
Tabuleiro 189 invalido
Tabuleiro 190 invalido
Tabuleiro 191 em andamento [X: sem jogada mestre]
Tabuleiro 192 em andamento [X: sem jogada mestre]
Tabuleiro 193 invalido
Tabuleiro 194 em andamento [X: sem jogada mestre]
Tabuleiro 195 em andamento [proximo jogador indefinido]
Tabuleiro 196 em andamento [proximo jogador indefinido]
Tabuleiro 197 em andamento [X: sem jogada mestre]
Tabuleiro 198 em andamento [proximo jogador indefinido]
Tabuleiro 199 em andamento [X: sem jogada mestre]
Tabuleiro 200 invalido
